src/main/java/org/jitsi/videobridge/VideoTransformChainManipulator.java

```diff
@@ -0,0 +1,24 @@
+package org.jitsi.videobridge;
+
+import org.jitsi.impl.neomedia.transform.TransformEngine;
+
+/**
+ * An interface for video transform chain manipulation
+ *
+ * You can register VideoTransformChainManipulator through OSGi.
+ *
+ * @author Laszlo Luczo
+ * @author Norbert Papp
+ */
+public interface VideoTransformChainManipulator
```

Comment:
Can we rename this to TransformChainManipulator? I don't see any video-specific code.